src/style.css

```diff
@@ -0,0 +1,48 @@
+.container-1 input#search {
```

Comment:
don't stylize tags and don't increase the specificity, use direct classes instead

Reply:
fix it everywhere